src/proto/message_translation/tbots_protobuf.cpp

```diff
@@ -30,10 +34,15 @@ std::unique_ptr<TbotsProto::World> createWorldWithSequenceNumber(
     *(world_msg->mutable_ball())          = *createBall(world.ball());
     *(world_msg->mutable_game_state())    = *createGameState(world.gameState());
     world_msg->set_sequence_number(sequence_number);
-
+    if (world.getDribbleDisplacement().has_value()) {
+        *(world_msg->mutable_dribble_displacement()) = *createSegmentProto(world.getDribbleDisplacement().value());
+    } else {
+        world_msg->clear_dribble_displacement();
+    }
```

Comment:
I don't think you need the else case. I believe the default will set this as cleared when the `world_msg` object is created

Reply:
bump